### PR TITLE
core: page function updates relating to upcoming tap targets audit

### PR DESCRIPTION
**Pull request by mattzeunert**
**Summary**

Splitting up [this PR](https://github.com/GoogleChrome/lighthouse/pull/5846) into more digestible bits. 

This PR just contains some page function tweaks, and a new function to get an element selector that we can show to the user.

**Related Issues/PRs**

Tap targets ticket: https://github.com/GoogleChrome/lighthouse/issues/4358

### CODE REVIEW

File: lighthouse-core/test/lib/page-functions-test.js [hunk 1]
```diff
@@ -44,5 +44,10 @@ describe('DetailsRenderer', () => {
         ['style-missing', 'aria-label-missing']
       ), '<div id="1" style="style" aria-label="label">');
     });
+
+    it('works if attribute values contain line breaks', () => {
```
Comment (patrickhulce) [inline lighthouse-core/test/lib/page-functions-test.js]: ❤️

File: lighthouse-core/lib/page-functions.js [hunk 2]
```diff
@@ -153,6 +153,71 @@ function ultradumbBenchmark() {
...
+  path.reverse();
+  return path.join(',');
+}
+
+/**
+ * @param {Element} node
+ * @returns {string}
+ */
+/* istanbul ignore next */
+function getNodeSelector(node) {
```
Comment (patrickhulce) [inline lighthouse-core/lib/page-functions.js]: think we could get a quick test for this function? don't need crazy coverage just something to make sure we don't break it

File: lighthouse-core/gather/gatherers/accessibility.js [hunk 3]
```diff
@@ -45,6 +45,7 @@ function runA11yChecks() {
     // Augment the node objects with outerHTML snippet & custom path string
     // @ts-ignore
     axeResult.violations.forEach(v => v.nodes.forEach(node => {
+      // @ts-ignore - getNodePath put into scope via stringification
       node.path = getNodePath(node.element);
```
Comment (Hoten) [inline lighthouse-core/gather/gatherers/accessibility.js]: you could keep this the same, if you also export `getNodePath` in `page-functions.js`.

Will need that module to export `getNodePath` for linking nodes in the report for the font-size audit (#6436) (and for other non-accessibility audits).

Comment (patrickhulce) [reply lighthouse-core/gather/gatherers/accessibility.js]: I'm not sure what you mean @Hoten, it's run in the context of the page and he's exported the stringified version which is what he'll need, no?

Comment (Hoten) [reply lighthouse-core/gather/gatherers/accessibility.js]: Didn't realize all of that was running in context of the page. Nevermind!

**Pull request merged**